script/push_docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright 2021 Adevinta
+
+# This script tags and pushes a locally existent docker image to a specified new
+# tag. It receives two parameters, the first one specifies the tag of the local
+# image and the second one the tag to push the image to. It expects the env
+# variables: DOCKER_USERNAME and DOCKER_PASSWORD to be set. Those credentials
+# must belong to a valid user with permission to push to the docker image
+# registry the script will push the image to.
+
+set -ev
```

Comment:
I'd also add `-u`. Why do you need `-v`?

Reply:
Yes "-u" it's a good option for catching errors when unset variables are referenced I'm adding it.
Regarding "-v" it's there because was convenient while testing the script, now that we ara close to have a first version I agree it is not needed anymore.